diff --git a/node_modules/@statechannels/devtools/lib/src/config/env.js b/node_modules/@statechannels/devtools/lib/src/config/env.js
index b54571f..b8e3bc0 100644
--- a/node_modules/@statechannels/devtools/lib/src/config/env.js
+++ b/node_modules/@statechannels/devtools/lib/src/config/env.js
@@ -30,10 +30,16 @@ exports.getEnvBool = getEnvBool;
 function configureEnvVariables() {
     const SC_ENV = process.env.SC_ENV;
     if (SC_ENV) {
-        const scEnvFile = path.join('../..', '.env.' + SC_ENV);
-        const scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
+        // node_modules/@statechannels/($simple-hub|$xstate-wallet)
+        let scEnvFile = path.join('../../..', '.env.' + SC_ENV);
+        let scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
         if (!fs.existsSync(scEnvFileFullPath)) {
-            throw new Error(`${scEnvFileFullPath} must exist in the monorepo root`);
+            // packages/$app/<xyz>
+            scEnvFile = path.join('../..', '.env.' + SC_ENV);
+            scEnvFileFullPath = path.join(process.cwd(), scEnvFile);
+            if (!fs.existsSync(scEnvFileFullPath)) {
+                throw new Error(`${scEnvFileFullPath} must exist in the monorepo root`);
+            }
         }
         const result = require('dotenv-expand')(require('dotenv').config({
             path: scEnvFile
